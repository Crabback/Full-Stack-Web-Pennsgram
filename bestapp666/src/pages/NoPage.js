const NoPage = () => {
    return <h1>404 not found</h1>;
  };
  
  export default NoPage;