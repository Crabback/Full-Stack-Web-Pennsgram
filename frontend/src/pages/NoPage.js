import React from 'react';
import Alert from 'react-bootstrap/Alert';

function NoPage(){
    return (
    <Alert id="noPageID" variant="danger" >
      <Alert.Heading>404. Page Not Found. </Alert.Heading>
      <p>
        Change this and that and try again. 
      </p>
  </Alert>)
  };
  
  export default NoPage;