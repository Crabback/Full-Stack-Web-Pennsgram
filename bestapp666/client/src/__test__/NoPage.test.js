/** @jest-environment jsdom */

import React from "react";
import { render, screen } from "@testing-library/react";
import "@testing-library/jest-dom/extend-expect";
import userEvent from "@testing-library/user-event";
import { BrowserRouter } from "react-router-dom";
import NoPage from "../pages/NoPage";

test("render friends component", () => {
  const test = render(
      <NoPage />
  );
  const element = test.getByText(/404/);
  expect(element).toBeVisible();
});
