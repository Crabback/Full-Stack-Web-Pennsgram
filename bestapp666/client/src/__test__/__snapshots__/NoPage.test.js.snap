// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render friends component 1`] = `
<div
  className="fade alert alert-danger show"
  id="noPageID"
  role="alert"
>
  <div
    className="alert-heading h4"
  >
    404. Page Not Found. 
  </div>
  <p>
    Change this and that and try again.
  </p>
</div>
`;
